zaptest/logger_test.go

```diff
@@ -105,5 +153,9 @@ func (t *testLogSpy) Logf(format string, args ...interface{}) {
 }
 
 func (t *testLogSpy) AssertMessages(msgs ...string) {
-	assert.Equal(t, msgs, t.Messages)
+	assert.Equal(t.TB, msgs, t.Messages, "logged messages did not match")
+}
+
+func (t *testLogSpy) AssertFailed(v bool) {
```

Comment:
Can we make this an unexported method and have methods `AssertFailed()` and `AssertPassed()` rather than a boolean argument?

Reply:
Makes sense